draft-ietf-httpbis-sfbis.md

```diff
@@ -191,6 +191,9 @@ being used.</t>
 </artwork>
 </blockquote>
 
+Note that a field is defined by referencing a specific RFC for Structured Fields, and its value is limited to the types defined in that RFC. For example, a field whose definition references this document can use the Date type ({{date}}), whereas a field whose definition references RFC8941 cannot. This also limits the kinds of extension values that can be later defined for a given field.
```

Comment:
I might add:

> Otherwise, an extension that uses a type that is permitted by a newer version of the Structured Field specification might be treated as invalid (and discarded) by an implementation that only implements an older version.

I don't know if you want to say that there is a possibility that though an extension like this would be accepted by those implementations that support the newer version, that's not consistently interoperable.

Note also this this doesn't just apply to types.  It might apply if the parsing/validation rules for a type becomes more lenient in a subsequent version.

Reply:
See revision